.travis.yml

```diff
@@ -138,6 +138,17 @@ jobs:
            - arduino --board esp8266:esp8266:generic:xtal=80,eesz=1M64,FlashMode=qio,FlashFreq=40,dbg=Serial,lvl=CORE --save-prefs
      after_success:
            - mv $PWD/release/SIGNALDuino.ino.bin $PWD/release/SIGNALDuino_${BOARD}${RECEIVER}${TRAVIS_TAG}.hex
+    - 
+     stage: deploy
+     env:
+       - BOARD=MAPLEMINI_F103CB
```

Comment:
Dann Compilieren wir nur die Version mit cc1101 und nicht die mit einem "einfachen" Empfänger/Sender?

Im plattformIO habe ich auch die anderen Varianten gefunden. Soll das so sein oder kommt da noch was?

Reply:
Ich wollte erstmal sehen ob es funktioniert die Einstellungen und wenn ja, dann wird es vervollständigt.
Ungern mache ich mehrere Dinge auf einmal wenn ich nicht weiß ob es überhaupt funktioniert *smile*

Reply:
ich strate mal den fehhaften job im debug

Reply:
Ich denke die Struktur nun zu durchschauen. Mal sehen ob es richtig war beim letzten mal.
Alternativ muss der Boardname geändert werden auf GenF1. So steht es in der Board.txt.
Ergebnis mal abwarten.

Reply:
@sidey79 ich denke so
`- arduino --board STM32:cortex-m3:arduino:dfuoMethod=Maple DFU Bootloader original --save-prefs`

bei der Angabe ist jeweils immer board,mcu,core notwendig wenn ich alles richtig verstehe.

Reply:
@sidey79 
```
GenF1.build.core=arduino
GenF1.build.board=GenF1
GenF1.build.mcu=cortex-m3
```

Reply:
Laut Commandhelp ist die Syntax für --board wie folgt:
`er sollte in der Form "package:arch:board" oder "package:arch:board:options" sein`

Reply:
Du kannst dich auf den Job der im debug mode via ssh verbinden und das direkt testen :)

Reply:
> Du kannst dich auf den Job der im debug mode via ssh verbinden und das direkt testen :)

Dumm gefragt, WIE? Das würde ich das erste mal machen. *smile*

Reply:
`arduino --board STM32:stm32:GenF1 --save-prefs ` bringt keine Fehler aber das compilieren klappt nicht, weil wichtige Macros nur vorhanden sind wenn MAPLEMINI_103FC definiert wurde

Der Job ist hier
https://travis-ci.com/github/RFD-FHEM/SIGNALDuino/jobs/356070597

Reply:
Das Board sollte eigentlich auch in der boards.txt enthalten sein:
https://github.com/stm32duino/Arduino_Core_STM32/blob/b12a55485250076e872fa286a925d6b3ae8d3016/boards.txt#L1132